packages/client/src/explore/Explore.ts

```diff
@@ -18,6 +23,22 @@ export class Explore {
     this.authentication = authentication;
   }
 
+  /**
+   * Explore publications
+   *
+   * @param request - Request object for the query
+   * @param observerId - Optional id of a profile that is the observer for this request
+   * @returns Array of {@link PublicationFragment} wrapped in the {@link PaginatedResult} helper
```

Comment:
nit: 
```suggestion
   * @returns Array of {@link PublicationFragment} wrapped in {@link PaginatedResult}
```

IMO "helper" suggests a instance of something or a function but it's just a type.

If you decide to action this, there are several occurrences of this text.